sdk/batch/azure-compute-batch/src/test/java/com/azure/compute/batch/JobTests.java

```diff
@@ -210,5 +217,48 @@ public void canCRUDJobWithPoolNodeCommunicationMode() throws Exception {
         }
     }
 
-
+    @Test
+    public void testDeserializationOfBatchJobStatistics() throws IOException {
+        // Simulated JSON response with numbers as strings
+        String jsonResponse = "{"
+            + "\"url\":\"https://example.com/stats\","
+            + "\"startTime\":\"2022-01-01T00:00:00Z\","
+            + "\"lastUpdateTime\":\"2022-01-01T01:00:00Z\","
+            + "\"userCPUTime\":\"PT1H\","
+            + "\"kernelCPUTime\":\"PT30M\","
+            + "\"wallClockTime\":\"PT1H30M\","
+            + "\"readIOps\":\"1000\","
+            + "\"writeIOps\":\"500\","
+            + "\"readIOGiB\":0.5,"
+            + "\"writeIOGiB\":0.25,"
+            + "\"numSucceededTasks\":\"10\","
+            + "\"numFailedTasks\":\"2\","
+            + "\"numTaskRetries\":\"3\","
+            + "\"waitTime\":\"PT10M\""
+            + "}";
+
+        // Deserialize JSON response using JsonReader from JsonProviders
+        try (JsonReader jsonReader = JsonProviders.createReader(new StringReader(jsonResponse))) {
+            BatchJobStatistics stats = BatchJobStatistics.fromJson(jsonReader);
+
+            // Assertions
+            Assertions.assertNotNull(stats);
+            Assertions.assertEquals("https://example.com/stats", stats.getUrl());
+            Assertions.assertEquals(OffsetDateTime.parse("2022-01-01T00:00:00Z"), stats.getStartTime());
+            Assertions.assertEquals(OffsetDateTime.parse("2022-01-01T01:00:00Z"), stats.getLastUpdateTime());
+            Assertions.assertEquals(Duration.parse("PT1H"), stats.getUserCpuTime());
+            Assertions.assertEquals(Duration.parse("PT30M"), stats.getKernelCpuTime());
```

Comment:
I added assertions to check the values of all of the properties in every class, but that might be overkill—Would it be better to just assert the values of the long properties only? _(e.g. only readIOps, writeIOps, numSucceededTasks, numFailedTasks, numTaskRetries in this case)_